src/mpid/ch4/src/ch4i_comm.h

```diff
@@ -1153,6 +1153,22 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_comm_create_rank_map(MPIR_Comm * comm)
         MPIDI_COMM(comm, local_map).mode = MPIDI_RANK_MAP_NONE;
     }
 
+#ifdef MPL_USE_DBG_LOGGING
+    int rank_;
+    int avtid_, lpid_;
+    if (comm->remote_size < 16) {
+        for (rank_ = 0; rank_ < comm->remote_size; ++rank_) {
+            MPIDIU_comm_rank_to_pid(comm, rank_, &lpid, &avtid);
```

Comment:
did you mean to use the underscore versions of `lpid` and `avtid` variables here?